src/ViewerComponents/Resource/SingleResource.svelte

```diff
@@ -46,8 +39,7 @@
             }
             // Requests the resource
             const iri = search ? search['@id'] : resource['Id'];
```

Comment:
It feels like this is two components mashed into one, depending on how it was reached. Eventually maybe we should sepereate them, looks fine for me at the moment though

Reply:
I also don't like this but I would also leave it that way for the moment. At least it works ;-)